dev/TreeView/TreeView.cpp

```diff
@@ -258,6 +258,21 @@ void TreeView::OnListControlDragItemsCompleted(const winrt::IInspectable& sender
 {
     auto treeViewArgs = winrt::make_self<TreeViewDragItemsCompletedEventArgs>();
     treeViewArgs->DragItemsCompletedEventArgs(args);
+
+    auto items = args.Items();
```

Comment:
>auto [](start = 4, length = 4)

per [https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#con4-use-const-to-define-objects-with-values-that-do-not-change-after-construction](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#con4-use-const-to-define-objects-with-values-that-do-not-change-after-construction) These locals should be defined as constant whenever possible.